container-images/tcib/base/os/neutron-base/neutron-server/neutron-server.yaml

```diff
@@ -1,5 +1,9 @@
 tcib_actions:
 - run: dnf -y install {{ tcib_packages['common'] | join(' ') }} && dnf clean all && rm -rf /var/cache/dnf
+# TODO(slaweq): install python3-neutron-fwaas on all distros once upstream
+# CI jobs will be switched to CentOS 10 Stream and this package from master
+# branch will be available there,
+- run: if [ '{{ tcib_distro }}' == 'rhel' ]; then dnf -y install python3-neutron-fwaas; fi
```

Comment:
can you also add as above && dnf clean all && rm -rf /var/cache/dnf as per previous comment